kit/preprocessors/hashInCode.js

```diff
@@ -6,12 +6,17 @@ export const hashInCodePreprocess = {
 	markup: async ({ content }) => {
 		const REGEX_CODE_BLOCK = /```.*?```/gs;
 		const REGEX_INLINE_CODE = /`.*?`/g;
+		const REGEX_INLINE_EXCEPTION = /(?!.*HF_DOC_BODY_START)`.*?`(?<!HF_DOC_BODY_END.*)/gs;
```

Comment:
logic tl;dr: find all code blocks (``.*?``) between `HF_DOC_BODY_START` and `HF_DOC_BODY_END`, the autodocs delimiters